arangopipe/tests/AQL/Running_AQL.md

```diff
@@ -0,0 +1,81 @@
+# Running AQL queries  on Arangopipe
+
+Apart, from using the **Arangopipe** search feature, it is also possible to query machine learning meta-data using the query feature in **ArangoDB**. An excerpt of the feature is shown in the figure below.
+<figure>
+  <img src="AQL_feature.png" alt="my alt text" width=90%/>
+  <figcaption style ="text-align:center">Figure 1.</figcaption>
+</figure>
+The left pane shows an editor area where we can enter queries. We can the execute the query by clicking on the *_Execute_* button in the lower right hand corner. Let us illustrate this with an example. The dataset used for illustration is the wine dataset used to illustrate how **Arangopipe** can be used with *_mlflow_* (see the *_mlflow_* folder). We will illustrate the following operations:
+
+1. Deleting a dataset from the database
+2. Deleting a featureset from the database
+3. Inserting a dataset into the database
+4. Inserting a featureset into the database
+5. Linking the featureset and the dataset
+
+If you would like to learn more about AQL, please view the [AQL documentation](https://www.arangodb.com/docs/stable/aql/). The [tutorials](https://www.arangodb.com/docs/stable/tutorials.html) section of the documentation may also be of interest.
+
+## Deleting a dataset from the database
+
+The first AQL operation that is demonstrated is the deletion of a *_dataset_* vertex from the **_dataset_** vertex collection. In particular, we will delete the *_dataset_* with the *_name_* attribute of "_wine dataset_". The AQL query for this operation is shown below.
+```
+
+FOR d IN datasets
+
+    FILTER d.name == 'wine dataset'
+  
+    REMOVE d IN datasets
+  
+```
+This query can be executed by entering the query in the query editor and execute the query by clicking the **_Execute_** button (see Figure 1).
+
+## Deleting a featureset from the database
+
+A featureset can be deleted from the database in a similar manner. The AQL query to delete a featureset vertex with a particular name is shown below.
+
+```
```

Comment:
Dito

Reply:
No, problem. I will move this also to the end.